src/modules/Dashboard/pages/Applet/AppletMultiInformant.tsx

```diff
@@ -37,7 +39,37 @@ export const AppletMultiInformant = () => {
     <StyledBody>
       {isLoading && <Spinner />}
       {appletData && (
-        <LinkedTabs hiddenHeader={hiddenHeader} tabs={appletTabs} isCentered={false} />
+        <>
+          <Box
+            display="flex"
+            flexDirection="row"
+            alignItems="center"
+            gap={theme.spacing(1.6)}
+            margin={`${theme.spacing(1.2)} ${theme.spacing(3.4)}`}
+          >
+            {appletData.image && (
+              <img src={appletData.image} width={48} style={{ borderRadius: '4px' }} />
+            )}
+            <StyledHeadlineLarge>{appletData.displayName}</StyledHeadlineLarge>
+            <Tooltip
+              title={appletData.description as string}
```

Comment:
Is it possible for appletData.description to be empty? Should we hide this icon/tooltip in that case?

Reply:
+1